Dockerfile

```
@@ -1,9 +1,9 @@
-ARG ruby_version=3.2
+ARG ruby_version=3.2.4
```

Best not to specify the patch version here, otherwise you'll prevent it from staying automatically up-to-date in future.
```suggestion
ARG ruby_version=3.2
```